src/shared/infrastructure/api/v1/country_routes.py

```
@@ -0,0 +1,122 @@
+
+    Returns:
+        Lista de países adyacentes ordenados alfabéticamente
+
+    Raises:
+        404: Si el país no existe
+    """
+    async with uow:
+        # Validar que el país existe
+        try:
+            code = CountryCode(country_code.upper())
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)